### DISCUSSION TITLE
Having issues creating my first itwin

### DISCUSSION THREAD

**Discussion by noblelar** [iTwin/itwinjs-core#6649] [Q&A]
**Please help me rectify this issue.** 

![Screenshot 2024-04-19 021206](https://github.com/iTwin/itwinjs-core/assets/59696875/0ecddffe-ab8a-4f5e-8b08-09f2789d4f42)

**Comment by williamkbentley** [comment]
what is the issue?

**Reply by noblelar** [reply]
My attempt to create an itwin has been unsuccessful.
An the note provided hasn't been helpful in fixing the situation 
Some assistance on how to go about the issue will be appreciated

**Reply by williamkbentley** [reply]
I'm guessing you're attempting to fill the form and give a test itwin a name. when you click save a toast message appears?

try refreshing the page.
try logout/login. 
try opening the network tab in devtools and find the failing request

**Reply by noblelar** [reply]
Yeah, I went back to delete an app I created, came back, refreshed, tried again, and my itwin has been successfully created. 
Thank you. For your assistance. It is very much appreciated .
